appcenter-post-build.sh

```diff
@@ -1,4 +1,41 @@
-# Script is to be included in the root directory of your repository. 
-curl -H "Content-Type: application/json" -H "authToken: 349ab29a-xtab-423b-a5hc-5623bc39b8c8" --data '{}' https://api.example.com/endpoint
+# Sample post-build script for Github PR checks
```

Comment:
Should use something like `git branch --no-color 2> /dev/null | sed -e '/^[^*]/d' -e "s/* \(.*\)/\1$(parse_git_dirty)/"` to get the current branch for building the target_url. Let's use another build environment var for the owner/app name in the App Center path.

Reply:
Got it, will do after adding support for multiple repos iteration/refactoring.